Sources/Datadog/DDRUMMonitor.swift

```diff
@@ -79,13 +79,15 @@ public class DDRUMMonitor {
     /// Notifies that an Error occurred in currently presented View.
     /// - Parameters:
     ///   - message: a message explaining the Error.
+    ///   - type: the type of Error.
     ///   - source: the origin of the error.
     ///   - stack: stack trace of the error. No specific format needed. Overwrites `file` and `line` parameters below.
     ///   - attributes: custom attributes to attach to the Error
     ///   - file: the file in which the Error occurred (the default is the file name in which this method was called).
     ///   - line: the line number on which the Error occurred (the default is the line number on which this method was called).
     public func addError(
         message: String,
+        type: String?,
```

Comment:
Thank you for your contribution @AvdLee 🙏 
For backward compatibility, it would be best to add `nil` as a default value here.